Cron/Order.php

```diff
@@ -225,19 +225,21 @@ protected function cancelOrder(OrderInterface $order) : bool
     protected function approveOrder(OrderInterface $order) : bool
     {
         try {
-            if ($order->getState() != MagentoOrder::STATE_HOLDED) {
-                return true;
+            if ($order->getState() == MagentoOrder::STATE_HOLDED) {
```

Comment:
LGTM. The only thing I would suggest would be to use `===` for type checking (especially since that's what you're doing below) but not a big deal really.

Reply:
Updated, @davehensleyns8. Thanks